third_party/wabt/src/interp/jit/jit-backend.cc

```diff
@@ -260,6 +260,26 @@ static void emitUnary(sljit_compiler* compiler, Instruction* instr) {
     case Opcode::I64Popcnt:
       // Not supported yet.
       return;
+    case Opcode::I32WrapI64:
+      sljit_emit_op1(compiler, SLJIT_MOV, SLJIT_R0, 0, args[0].arg,
+                     args[0].argw);
+      sljit_emit_op1(compiler, SLJIT_MOV32, args[1].arg, args[1].argw, SLJIT_R0,
+                     0);
+      return;
+    case Opcode::I64ExtendI32S:
+      sljit_emit_op1(compiler, SLJIT_MOV, SLJIT_R0, 0, SLJIT_IMM, -1);
```

Comment:
What is the purpose of this code? It is essentially a dead code.

Reply:
You're right, It was left over from earlier debugging.